src/Oscar.jl

```diff
@@ -87,6 +87,7 @@ function __init__()
         (GAP.Globals.IsFpGroup, FPGroup),
     ])
     GAP.Packages.load("forms")
+    GAP.Packages.load("ctbllib")
```

Comment:
What's this about? Is this preparation for moving your character table code into the "official" part of Oscar.jl (you are still working on that, right?)

Reply:
Yes, this addition accidentally slipped in ...